Classes/Models/File+Filename.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  File+Filename.swift
+//  Freetime
+//
+//  Created by Weyert de Boer on 13/10/2017.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+
+extension File {
+
+    /// Returns the potential programming language of the file, if unsure it will return `nil`
+    var potentialLanguage: String? {
```

Comment:
Not sure this is necessary, at least yet? Also it shouldn't **only** support Swift, even if we do keep it